h2o-core/src/main/java/water/rapids/ast/prims/search/AstMatch.java

```diff
@@ -23,13 +21,13 @@
 public class AstMatch extends AstPrimitive {
   @Override
   public String[] args() {
-    return new String[]{"ary", "table", "nomatch", "incomparables"};
```

Comment:
I try to find how incomparables parameter works in the base library. Besides the base documentation I found this thread where people struggled with the parameter. It seems it is not working as the documentation explains. 

https://stat.ethz.ch/R-manual/R-devel/library/base/html/match.html

IMO this feature is useless. So I decided not to make a PR for it.  WDYT?

Reply:
this param was actually unused by our implementation anyway, right? so better remove it then, thanks.

Reply:
weird that R failed implementing it correctly though, doesn't look that complicated (maybe it's fixed in more recent versions, didn't check)

Reply:
yes it is :)

Reply:
In the h2o.match function, we introduced this parameter, but was not working anymore and no one complained about it. :woman_shrugging:

Reply:
sure, I still don't mind removing it: better no param than not working param.

Just noticing that it's been fixed on R side since.